xchange-coingi/src/main/java/org/knowm/xchange/coingi/dto/request/WithdrawalRequest.java

```diff
@@ -0,0 +1,36 @@
+package org.knowm.xchange.coingi.dto.request;
```

Comment:
org.knowm.xchange.coingi.dto.account

Reply:
All HTTP Request DTOs have their own package (easier maintenance). I don't understand the suggestion here for the WithdrawalRequest (it's a HTTP Request, not your typical DTO) .

Reply:
moved